### PR TITLE
Add big-endian, s390x job for Travis

### PR DESCRIPTION
**Pull request by jamessan**
This will help catch issues like #11388

### CODE REVIEW

**Comment by bfredl** [comment]
Hmm I can't access travis right now. Will try again later.

File: .travis.yml [hunk 1]
```diff
@@ -160,6 +159,26 @@ jobs:
...
+        apt:
+          packages:
+            - *common-apt-packages
+            - gettext
+            - python-pip
+            - python3-pip
+            - python-setuptools
+            - python3-setuptools
+            - python-dev
+            - python3-dev
```
Comment (blueyed) [inline .travis.yml]: Is Python not installed there by default?

File: ci/install.sh [hunk 2]
```diff
@@ -14,9 +14,9 @@ fi
 
 # Use default CC to avoid compilation problems when installing Python modules.
 echo "Install neovim module for Python 3."
-CC=cc python3 -m pip -q install --upgrade pynvim
+CC=cc python3 -m pip -q install --user --upgrade pynvim
 echo "Install neovim module for Python 2."
-CC=cc python2 -m pip -q install --upgrade pynvim
+CC=cc python2 -m pip -q install --user --upgrade pynvim
```
Comment (blueyed) [inline ci/install.sh]: IIRC using `--user` might end up with binaries not being available in the `$PATH` - but should be fine for the lib.

**Review by blueyed — COMMENTED** [review]
LGTM in general.

**Review by bfredl — APPROVED** [review]

File: ci/install.sh (hunk 2 above)
Comment (jamessan) [reply ci/install.sh]: There were [permission issues](https://travis-ci.com/github/jamessan/neovim/jobs/299690353#L463) when not using `--user`.  Would it be preferable to use `sudo` here instead?

File: .travis.yml (hunk 1 above)
Comment (jamessan) [reply .travis.yml]: Doesn't look like it.  The build [failed to find `Python.h`](https://travis-ci.com/github/jamessan/neovim/jobs/299695174#L456) before I added these.

File: ci/install.sh (hunk 2 above)
Comment (blueyed) [reply ci/install.sh]: No, I prefer `--user` over `sudo` if possible.

**Review by blueyed — APPROVED** [review]

**Pull request merged**